src/views/LoginView.vue

```diff
@@ -60,7 +60,7 @@
                 class="mt-4"
                 variant="solo"
                 :rules="usernameRules"
-                :dense="$vuetify.display.mdAndDown ? true : false"
+                :density="$vuetify.display.mdAndDown ? true : false && 'compact'"
```

Comment:
I think these are not correct. The general idea is that the `dense` prop was boolean (true meant smaller) and it's now replaced by a `density` prop which takes a string value of `'compact'` or `'default'`. We want this to be compact on small devices so what we actually want is
```suggestion
                :density="$vuetify.display.mdAndDown ? 'compact' : false && 'default'"
```